content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -30,9 +30,35 @@ To enable debug mode for the Datadog Java Tracer, set the flag `-Ddd.trace.debug
 
 {{< programming-lang lang="python" >}}
 
-To enable debug mode for the Datadog Python Tracer, set the environment variables `DD_TRACE_DEBUG=true` and `DD_CALL_BASIC_CONFIG=true` when using `ddtrace-run`.
+The steps for enabling debug mode Datadog Python Tracer, depends on which version of the tracer your application is using. Choose the scenario that applies:
+
+### Scenario 1: ddtrace version 1.3.0 or higher
+
+1. Set the `DD_TRACE_DEBUG` environment variable to `true` to enable debug mode.
+
+2. To route debug logs to a log file, set `DD_TRACE_LOG_FILE` with a filename that tracer logs should be written to, relative to the current working directory. For example, `DD_TRACE_LOG_FILE=ddtrace_logs.log`. 
+   By default, the file size is 15728640 bytes (about 15MB) and one backup log file is created. To increase the default log file size, specify the size in bytes with the `DD_TRACE_LOG_FILE_SIZE_BYTES` setting.
+
+3. To route logs to the console, for **Python 2** applications, configure `logging.basicConfig()` or similar. Logs are automatically sent to the console for **Python 3** applications. 
+
+
+### Scenario 2: ddtrace version 1.0.x to 1.2.x
+
+1. To enable debug mode: `DD_TRACE_DEBUG=true`
+    - `DD_CALL_BASIC_CONFIG=true`
+
+2. To route logs to the console, for **Python 2 or Python 3** applications, configure logging.basicConfig() or use DD_CALL_BASIC_CONFIG=true.
+
+### Scenario 3: ddtrace version 0.x
+
+1. Set the `DD_TRACE_DEBUG` environment variable to `true` to enable debug mode.
+
+2. To route logs to the console, for **Python 2 or Python 3** applications, configure logging.basicConfig() or use DD_CALL_BASIC_CONFIG=true.
+
+### Scenario 4: Configuring debug logging in the application code with the standard logging library
+
+For any version of dd-trace-py, rather than setting the `DD_TRACE_DEBUG` tracer environment variable, you can enable debug logging in the application code by using the `logging` standard library directly:
```

Comment:
```suggestion
For any version of ddtrace, rather than setting the `DD_TRACE_DEBUG` tracer environment variable, you can enable debug logging in the application code by using the `logging` standard library directly:
```